src/main/java/org/gridsuite/filter/server/FilterService.java

```diff
@@ -65,41 +65,17 @@ public class FilterService {
     private final FilterService self;
 
     public FilterService(final ScriptFilterRepository scriptFiltersRepository,
-                         final LineFilterRepository lineFilterRepository,
```

Comment:
It's a pity to change just a part of the code. We should delete all the code relative to criteria filters

Reply:
yes but we will do it in another PR after the migration works and there is no problem we will do another PR where we will clean all the code. Same for commons-ui.